### PR TITLE
Controllers

### PR DESCRIPTION
**Pull request by yuvenalmash**
## Changes Summary
- Define the controller to:
  - Show all users
  - Show a single user
  - Show all posts for a user
  - Show a single post for a user
- Add the routes
- Add the views

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    @user = User.find(params[:id])
+  end
+
+  def index_posts
+    @posts = Post.where(user_id: params[:user_id])
+  end
+
+  def show_post
+    @post = Post.find(params[:id])
+  end
```
Comment (BregornOriginal) [inline app/controllers/users_controller.rb]: Good work creating a controller for users 💯 

As you create this controller for users, you need to create also a controller for posts. In this way, you will be able to have a different route and view for each controller.

- Kindly create a controller for post with the corresponding `index` and `show` methods as you did with user

File: app/views/users/index_posts.html.erb [hunk 2]
```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE HTML>
+<html>
```
Comment (BregornOriginal) [inline app/views/users/index_posts.html.erb]: - Kindly put your post files from your view, inside a folder for the posts views to follow the architecture MVC. It should look like `app/views/posts/index.html.erb` and `app/views/posts/show.html.erb`

**Review by BregornOriginal — CHANGES_REQUESTED** [review]
## Hi @yuvenalmash  

## Good job so far!

## There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

✅ Linters are passing
✅ Good commit history
✅ Good Pull Request description
✅ Good work creating routes for each case 🥇 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mukhammadrizooff — APPROVED** [review]
Hi @yuvenalmash, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- No linter errors ✔️ 
- Correct GitFlow ✔️ 
- Good implementations ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**